x-pack/plugins/security_solution/public/common/components/event_details/cti_details/risk_summary.tsx

```diff
@@ -12,69 +12,80 @@ import * as i18n from './translations';
 import { EnrichedDataRow, ThreatSummaryPanelHeader } from './threat_summary_view';
 import { RiskScore } from '../../severity/common';
 import type { RiskSeverity } from '../../../../../common/search_strategy';
-import type { HostRisk } from '../../../../risk_score/containers';
+import { RiskScoreEntity } from '../../../../../common/search_strategy';
+import type { HostRisk, UserRisk } from '../../../../risk_score/containers';
 import { getEmptyValue } from '../../empty_value';
 import { RiskScoreDocLink } from '../../risk_score/risk_score_onboarding/risk_score_doc_link';
-import { RiskScoreEntity } from '../../../../../common/search_strategy';
 import { RiskScoreHeaderTitle } from '../../risk_score/risk_score_onboarding/risk_score_header_title';
 
-const HostRiskSummaryComponent: React.FC<{
-  hostRisk: HostRisk;
-  originalHostRisk?: RiskSeverity | undefined;
-}> = ({ hostRisk, originalHostRisk }) => {
-  const currentHostRiskScore = hostRisk?.result?.[0]?.host?.risk?.calculated_level;
+interface HostRiskEntity {
+  originalRisk?: RiskSeverity | undefined;
+  risk: HostRisk;
+  riskEntity: RiskScoreEntity.host;
+}
+
+interface UserRiskEntity {
+  originalRisk?: RiskSeverity | undefined;
+  risk: UserRisk;
+  riskEntity: RiskScoreEntity.user;
+}
+
+export type RiskEntity = HostRiskEntity | UserRiskEntity;
+
+const RiskSummaryComponent: React.FC<RiskEntity> = ({ risk, riskEntity, originalRisk }) => {
+  const currentRiskScore =
+    riskEntity === RiskScoreEntity.host
```

Comment:
100% a personal nit, but I tend to lean `if` statements over nested ternaries because I think it's more readable and easily extended later on. 🤷🏾‍♂️ I know it's going on in a few other places here, but just commenting here.

Reply:
this is not a nested ternary but i can think of the spot you might be referencing... going to leave it as is here